### PR TITLE
docs: add WoodpeckerCI to showcase

### PR DESCRIPTION
**Pull request by 6543**
## Motivation

show how a modern documentation website could look like

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/docusaurus/blob/main/CONTRIBUTING.md#pull-requests)?

yes - there is not much about adding showcases

## Test Plan

well it's a showcase, best would be to look at the website :)

## Related PRs
none

### CODE REVIEW

File: website/src/data/users.tsx [hunk 1]
```diff
@@ -1535,6 +1535,14 @@ const Users: User[] = [
     source: 'https://github.com/whirl21/website',
     tags: ['opensource', 'personal'],
   },
+  {
+    title: 'WoodpeckerCI',
+    description: "Woodpecker is a simple CI engine with great extensibility.",
+    preview: require('./showcase/woodpecker.png'),
+    website: 'https://woodpecker-ci.org/',
+    source: 'https://github.com/woodpecker-ci/woodpecker/tree/master/docs',
```
Comment (Josh-Cena) [inline website/src/data/users.tsx:1543]:
```suggestion
    source: 'https://github.com/woodpecker-ci/woodpecker',
```

Comment (Josh-Cena) [reply website/src/data/users.tsx:1543]: No need for pointing to subdirectories :)

Comment (6543) [reply website/src/data/users.tsx:1543]: @Josh-Cena should I squash the fix?

Comment (Josh-Cena) [reply website/src/data/users.tsx:1543]: No need, we squash merge anyways. Thanks!

**Comment by Josh-Cena** [comment]
Thanks! As soon as the CLA check passes we can merge

**Comment by 6543** [comment]
@Josh-Cena :point_up: :tada:

**Comment by Josh-Cena** [comment]
👍

**Pull request merged**